draft-ietf-jose-hpke-encrypt.md

```diff
@@ -230,9 +229,10 @@ Where:
 
  - `×||y` is the concatenation of byte strings x and y.
  - len32(x) is number of bytes in x as four-byte big-endian integer.
- - enc is the value of "enc" header parameter in JOSE header.
+ - enc is the value of "enc" header parameter in JOSE header. The integrity-protected 'enc' parameter provides protection against an
+   attacker who manipulates the encryption algorithm in the 'enc' parameter.
  - apu is The value of "apu" header parameter if present in JOSE header, otherwise empty string.
- - apv is The value of "apv" header parameter if present in JOSE header, otherwise empty string.
+ - apv is The value of "apv" header parameter if present in JOSE header, otherwise empty string. 
```

Comment:
"apu" and "apv" should be removed as broken. And the recipient context structure above should be updated to something like:

`len32(enc)||enc`.